cloudrail/knowledge/context/mergeable.py

```
@@ -82,13 +84,18 @@ def origin(self) -> EntityOrigin:
         if self.is_pseudo:
             return EntityOrigin.PSEUDO
         elif self.iac_state:
-            return EntityOrigin.TERRAFORM
+            if self.iac_state.iac_type == IacType.TERRAFORM:
+                return EntityOrigin.TERRAFORM
+            elif self.iac_state.iac_type == IacType.CLOUDFORMATION:
+                return EntityOrigin.CLOUDFORMATION
+            else:
+                return EntityOrigin.NONE
```

Do we know what to do with NONE? Wouldn't it make more sense to throw an exception here?

---

i agree, None makes no sense and if we get None here for some reason we should raise an exception

---

@yi2020 @Tomerva done